includes/ppc-frontend.php

```diff
@@ -0,0 +1,93 @@
+<?php
+/**
+ * Pre-Publish Checklist php for Displaying the contents in the general tab in the plugin settings
+ * PHP version 7
+ *
+ * @category PHP
+ * @package  Pre-Publish Checklist.
+ * @author   Display Name <username@ShubhamW.com>
+ * @license  http://brainstormforce.com
+ * @link     http://brainstormforce.com
+ */
+
+// Displaying the contents in the general tab in the plugin settings.
+$ppc_radio_button        = get_option( 'ppc_radio_button_option_data' );
+$ppc_radio_button        = ( ! empty( $ppc_radio_button ) ? $ppc_radio_button : 3 );
+$ppc_checklist_item_data = get_option( 'ppc_checklist_data' );
+wp_enqueue_style( 'ppc_backend_css' );
+$ppc_post_types = get_option( 'ppc_post_types_to_display' );
+$ppc_args       = array(
+	'public' => true,
+);
+
+$ppc_exclude = array( 'attachment', 'elementor_library', 'Media', 'My Templates' );
+
+?>
+<!DOCTYPE html>
```

Comment:
The tabs are already inside `<html>`, This templates should not add the complete HTML markup, you can remove all the markup until `<form>`